src/components/SQForm/SQFormCheckbox.tsx

```diff
@@ -28,7 +42,7 @@ function SQFormCheckbox({
       <FormControlLabel
         control={
           <Checkbox
-            checked={field.value}
+            checked={field.value as boolean}
```

Comment:
Is there a way to know what type the value might be, rather than casting as `boolean`? We lose type safety with type assertions
It looks like the value is an array in SQFormCheckboxGroup, but I'm not sure how it's used elsewhere

Reply:
Cody and I talked offline - Changed to force it to boolean with `!!` - and did some looking at SQFormCheckboxGroup - that array value is never directly passed down to SQFormCheckbox